### DISCUSSION TITLE
Error 500 while uploading Plaso file

### DISCUSSION THREAD

**Discussion by s1lva-lucas** [google/timesketch#2183] [Q&A, quick solutions, support] [answered]
Hi everyone, 

I`m having the following error when trying to upload a new Plaso file to TimeSketch.

![image](https://user-images.githubusercontent.com/12535846/167743343-633a3d8d-617d-4ace-a289-b6b81a011592.png)

**Comment by jaegeral** [comment, marked as answer]
Hi, thanks for reporting, please use https://timesketch.org/guides/admin/troubleshooting/ as a guide to provide more information.

Cheers
